popup.js

```diff
@@ -206,10 +231,25 @@ function handleButtons () {
       addSkill();
     }
   });
+  $('#add_skill').click(addSkill);
+}
+
+function floor(num) {
```

Comment:
Remove this function. Try to use Math.floor() instead. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/floor